#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_free.1.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: -g flag documented #286900
## DP: hypen escaped #282168

@DPATCH@
diff -urNad procps-3.2.7~/free.1 procps-3.2.7/free.1
--- procps-3.2.7~/free.1	2004-02-24 08:35:55.000000000 +1100
+++ procps-3.2.7/free.1	2007-10-04 22:32:20.000000000 +1000
@@ -5,7 +5,7 @@
 .SH NAME
 free \- Display amount of free and used memory in the system
 .SH SYNOPSIS
-.BR "free " [ "\-b" " | " "\-k" " | " "\-m" "] [" "\-o" "] [" "\-s"
+.BR "free " [ "\-b" " | " "\-k" " | " "\-m" " | " "\-g" "] [" "\-o" "] [" "\-s"
 .I delay
 .RB "] [" "\-t" "] [" "\-V" ]
 .SH DESCRIPTION
@@ -13,32 +13,34 @@
 memory in the system, as well as the buffers used by the kernel.
 The shared memory column should be ignored; it is obsolete.
 .SS Options
-The \fB-b\fP switch displays the amount of memory in bytes; the 
-\fB-k\fP switch (set by default) displays it in kilobytes; the \fB-m\fP
-switch displays it in megabytes.
+The \fB\-b\fP switch displays the amount of memory in bytes; the 
+\fB\-k\fP switch (set by default) displays it in kilobytes; the \fB\-m\fP
+switch displays it in megabytes; the \fB\-g\fP switch displays it in
+gigabytes. 
 .PP
-The \fB-t\fP switch displays a line containing the totals.
+The \fB\-t\fP switch displays a line containing the totals.
 .PP
-The \fB-o\fP switch disables the display of a "buffer adjusted" line.
-If the -o option is not specified, \fBfree\fP subtracts buffer memory
+The \fB\-o\fP switch disables the display of a "buffer adjusted" line.
+If the \-o option is not specified, \fBfree\fP subtracts buffer memory
 from the used memory and adds it to the free memory reported.
 .PP
-The \fB-s\fP switch activates continuous polling \fIdelay\fP seconds apart. You
+The \fB\-s\fP switch activates continuous polling \fIdelay\fP seconds apart. You
 may actually specify any floating point number for \fIdelay\fP, 
 .BR usleep (3)
 is used for microsecond resolution delay times.
 .PP
 The \fB\-V\fP displays version information.
 .SH FILES
-.ta
-.IR /proc/meminfo "\-\- memory information"
-.fi
+.TP
+.I /proc/meminfo
+memory information
+
 
 .SH "SEE ALSO"
 .BR ps (1),
 .BR slabtop (1),
 .BR vmstat (8),
-.BR top(1)
+.BR top (1)
 
 .SH AUTHORS
 Written by Brian Edmonds. 
